CONTRIBUTING.md

```diff
@@ -14,14 +14,14 @@ We use a bunch of nice Open Source tools to make collaboration of both developer
 
 - We use [Node.js](https://github.com/nodejs/node) as JavaScript run-time environment to build the page.
```

Comment:
```suggestion
- We use [Node.js](https://nodejs.org/) as JavaScript run-time environment to build the page.
```
Should we link to the website instead of the repository?